### PR TITLE
fix get frames semple

### PR DESCRIPTION
**Pull request by NapalmRain**

### CODE REVIEW

File: PreviewThumbnailExtractor/Program.cs [hunk 1]
```diff
@@ -107,7 +107,11 @@ private static async Task ProcessThumbnailsAsync(string destination, Cancellatio
                     using (var image = new Image<SixLabors.ImageSharp.PixelFormats.Bgra32>((int)(Pitch / BytePerPixel), (int)Lines))
                     using (var sourceStream = file.file.CreateViewStream())
                     {
-                        sourceStream.Read(MemoryMarshal.AsBytes(image.GetPixelMemoryGroup().Single().Span));
+                        IMemoryGroup<SixLabors.ImageSharp.PixelFormats.Bgra32> mg = image.GetPixelMemoryGroup();
```
Comment (jeremyVignelles) [inline PreviewThumbnailExtractor/Program.cs:110]: You didn't set the using on top, so the build is failing. You could also use `var`

File: PreviewThumbnailExtractor/Program.cs [hunk 2]
```diff
@@ -107,7 +107,11 @@ private static async Task ProcessThumbnailsAsync(string destination, Cancellatio
...
+                        for(int i = 0; i < mg.Count; i++)
```
Comment (jeremyVignelles) [inline PreviewThumbnailExtractor/Program.cs:111]: Why not a foreach ?

**Review by jeremyVignelles — APPROVED** [review]
LGTM, didn't test

**Comment by mfkl** [comment]
Thanks!

**Pull request merged**